orga/tests/integration/components/ui/empty-state_test.js

```diff
@@ -0,0 +1,15 @@
+import { module, test } from 'qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+module('Integration | Component | Ui::EmptyState', function (hooks) {
+  setupIntlRenderingTest(hooks);
+
+  test('it should display a message telling so there is no participants', async function (assert) {
```

Comment:
**Suggestion :** _'it should display a message telling if there is no participants'_

Reply:
done